server/integration/testsuite/src/test/java/org/infinispan/server/test/dataconversion/Currency.java

```diff
@@ -0,0 +1,44 @@
+package org.infinispan.server.test.dataconversion;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.io.Serializable;
+
+import org.infinispan.commons.marshall.SerializeWith;
+
+/**
+ * @since 9.4
+ */
+@SerializeWith(value = Currency.Externalizer.class)
```

Comment:
SerializeWith does not exist. According to documentation :)))

Reply:
This is not actually needed, I could just use ```implements Serializable```, but adding this allowed me to test  deploying entities that depends on a internal infinispan module

Reply:
I was only joking. We can use that. It even works.